composer.json

```diff
@@ -7,7 +7,7 @@
     "magento/magento-composer-installer": "*"
   },
   "type": "magento2-module",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
Could you please completely remove the version (see https://blog.packagist.com/tagged-a-new-release-for-composer-and-it-wont-show-up-on-packagist/)?

Reply:
We've removed the version from the composer.json.